module/VuFind/src/VuFind/RecordDriver/SolrDefault.php

```diff
@@ -739,6 +739,11 @@ public function getLCCN()
         // Get LCCN from Index
         $raw = isset($this->fields['lccn']) ? $this->fields['lccn'] : '';
 
+        // First call number only.
```

Comment:
Was this added by accident? It doesn't seem related to Syndetics, and I don't think it should be necessary in core VuFind (but maybe it's related to one of your local customizations).

Reply:
This might be unique to us. I removed it and I'll push up a copy of this file without the array checking in my next push.